packages/flutter_tools/test/web.shard/test_data/expression_evaluation_web_common.dart

```diff
@@ -171,7 +171,10 @@ Future<void> testAll({required bool useDDCLibraryBundleFormat}) async {
         expressionEvaluation: expressionEvaluation,
         startPaused: true,
         script: project.testFilePath,
-        additionalCommandArgs: <String>['--verbose'],
+        additionalCommandArgs: <String>[
+          '--verbose',
+          if (useDDCLibraryBundleFormat) '--web-experimental-hot-reload',
```

Comment:
If possible can we add an `else '--no-web-experimental-hot-reload'`? That would make it smoother to enable the flag by default.

Reply:
Done!